Snowdog_Components/components/Molecules/table/_table.scss

```diff
@@ -84,6 +84,33 @@
         }
     }
 
+    &--tfoot-scope-row {
+        tfoot {
+            tr {
+                th {
+                    border-top: $table__tfoot-th-border-top;
```

Comment:
please modify this variable name so that it's specific for this variant, like the other variable